docs/Game/Unity/getting-started/01-connect.md

```diff
@@ -3,25 +3,56 @@ title: 01 - Connect Wallet and Authenticate
 id: game-01
 ---
 
+Connecting to a Web3 wallet such as MetaMask via **WalletConnect** provides a link between a **Wallet Address** and a user's **Game Account**. 
 
-Connecting to a Web3 i.e. MetaMask wallet via **WalletConnect** provides a link between a **Wallet Address** and a user's **Game Account**. 
+There are two ways to get a **Session Key** from your wallet. You can connect with `QRCode` or using `WalletConnect.Instance.OpenMobileWallet`. The easiest way to connect is with `QRCode`.
```

Comment:
Its not really the easiest way. It depends on the platform they want to build to.
QR Code should be used when building desktop-based applications
the Other way for android/ios 
we are working on a way for WebGL builds and standalone VR

Reply:
Ok. Amending this.